recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -23,91 +26,84 @@ class CubicInterpolationConan(ConanFile):
         "fPIC": True,
     }
 
-    generators = "cmake", "cmake_find_package"
     _cmake = None
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     @property
     def _is_msvc(self):
         return str(self.settings.compiler) in ["Visual Studio", "msvc"]
 
+    def export_sources(self):
+        #self.copy("CMakeLists.txt")
+        export_conandata_patches(self)
+
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
         # TODO: update boost dependency as soon as issue #11207 is fixed
```

Comment:
The mentioned issue is closed https://github.com/conan-io/conan-center-index/issues/11207

Can we update these? 🚀

Reply:
I believe this TODO note was inaccurate. The result of the discussion in issue #11207 was that we can update the boost dependency as soon as we deprecate conan1.x (since this means that `compiler.cppstd` will be mandatory by then).

I've clarified the todo note to specify this.

Reply:
🤔 Okay thats fine for now... I know this has changed many times but we can save that for another PR